spec/support/objects.rb

```diff
@@ -0,0 +1,43 @@
+module DummyApp
+  module Command
+    class Hello < Struct.new(:name)
+    end
+
+    class GoodBye < Struct.new(:name)
+    end
+
+    class NotHandlable < Struct.new(:name)
+    end
+  end
+end
+
+module DummyApp
+  module CommandHandler
+    class Hello
+      def handle(command)
+        "hello #{command.name}"
+      end
+    end
+
+    class GoodBye
+      def handle(command)
+        "good bye #{command.name}"
+      end
+    end
+  end
+end
+
+module DummyApp
+  class CommandResolver
+    def resolve(command)
+      handler_class(command).new
+    rescue NameError
+      nil
+    end
+
+    def handler_class(command)
+      klass = command.class.to_s.gsub('Command', 'CommandHandler')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.